minecraft_servers/kingdomblocks/manifest.json

```diff
@@ -0,0 +1,20 @@
+{
+  "server_name": "kingdomblocks",
+  "nice_name": "Kingdomblocks Network",
+  "direct_ip": "kingdomblocks.net",
+  "server_wildcards": [
+    "%.kingdomblocks.net"
+  ],
+  "social": {
+    "web": "https://kingdomblocks.net",
+    "web_shop": "https://shop.kingdomblocks.net",
+    "web_support": "https://kingdomblocks.net/forum",
+    "discord": "https://kingdomblocks.net/discord"
```

Comment:
As custom discord invite links are only supported for LabyMod partners at the moment.
Please use a discord invite link.

Documentation reference: https://github.com/LabyMod/server-media/blob/master/docs/Manifest.md#discord-url